docs/beginners-guide/keybinds-overview.md

```diff
@@ -15,6 +15,8 @@ Also see:
 
 **TIP:** `<M>` is `alt` on Windows/Linux and `option` on MacOS
 
+**TIP:** If you happen to be using iTerm2, take note that `<M>` requires the `option` key to be mapped to `Esc+` in Preferences - Profiles - Keys for proper functioning in both the terminal and tmux
```

Comment:
You are a lifesaver thank you for this. Here is a screenshot of the gui option if anyone else is searching for this 
![image](https://user-images.githubusercontent.com/959505/226060107-3310320d-637a-4d2a-b877-1cf524dc83d2.png)